src-electron/generator/template-util.js

```diff
@@ -229,11 +256,10 @@ async function ensureTemplatePackageId(context) {
  * @returns endpoint type ids
  */
 async function ensureEndpointTypeIds(context) {
-  let pkg = await queryPackage.getPackageByPackageId(
-    context.global.db,
-    context.global.genTemplatePackageId
-  )
-  let packageCategory = pkg.category
+  let packageCategory = null
+  if (context.global.genTemplatePackage != null) {
+    packageCategory = context.global.genTemplatePackage.category
```

Comment:
Should we be doing `packageCategory = await ensureTemplatePackageCategory(context)` ?